packages/medusa/src/api/routes/admin/sales-channels/index.ts

```diff
@@ -47,11 +48,19 @@ export default (app) => {
   )
   salesChannelRouter.post(
     "/stock-locations",
+    checkRegisteredModules({
+      stockLocationService:
+        "Stock Locations are not enabled. Please add a Stock Location module to enable this functionality.",
+    }),
```

Comment:
suggestion: store that into a const prior those routes and then apply in on the concerned routes